pkg/projects/provider.go

```diff
@@ -36,7 +36,7 @@ func Provider() *schema.Provider {
 		},
 
 		ResourcesMap: map[string]*schema.Resource{
-			"project": projectResource(),
+			"project_project": projectResource(),
```

Comment:
?? NO! It can't be that dumb

Reply:
I look up the HCL guideline/convention, it's `provider name_resource name` so unfortunately the way we current name our provide and resource, we ended up with `project_project` 😢

Reply:
That's dumb. It has to change.